### ISSUE TITLE
Feature request: simplify editing the list of web archive services

### ISSUE THREAD

**Issue by gvlx** [arantius/resurrect-pages#48]
Currently, adding a new web archive service implies a PR, changing, at least, four files (as per #46 ).

This extension should provide an option screen where the list of archive services can be changed.

**Comment by arantius** [comment]
I do not believe enough such services exist to make such a complication worthwhile.

**Comment by gvlx** [comment]
Currently, the selection of a particular web archive service is _hard-coded_ in, at least, four files in your code.

Should this information be placed in a json file, we could, in theory, manage the list much more easily.

The existing [list of country-specific web archive services is quite large](https://en.wikipedia.org/wiki/List_of_web_archiving_initiatives) and there's no guarantee they all use the same [api/protocol as the WayBackMachine](https://archive.org/help/wayback_api.php), so I understand this request is a rather heavy burden to you.

I would, however, insist that you should have look at the complexity of the change.

If the task is not insurmountable, maybe we could the interest of all those services in supporting your work?
